docs/training/apis-howto/api-pytorch-ug.rst

```diff
@@ -638,7 +636,271 @@ Take the following code for example.
+-  Load previous saved checkpoints directly into your model.
+
+Initializing the Trainer
+========================
+
+After defining the PyTorch Trial, initialize the trial and the trainer. ``pytorch.init()`` returns a
+``PyTorchTrialContext`` for instantiating ``PyTorchTrial``. Initialize ``pytorch.Trainer`` with the
+trial and context.
+
+.. code:: python
+
+   from determined import pytorch
+   def main():
+       with det.pytorch.init() as train_context:
+           trial = MyTrial(train_context)
+           trainer = det.pytorch.Trainer(trial, train_context)
+
+   if __name__ == "__main__":
+       # Configure logging
+       logging.basicConfig(level=logging.INFO, format=det.LOG_FORMAT)
+       main()
+
+Training is configured with a call to ``.fit()`` with training loop arguments, such as checkpointing
+periods, validation periods, and checkpointing policy. See the :doc:`Pytorch API reference document
+</reference/reference-training/training/api-pytorch-reference>` for the full list of arguments.
+
+.. code:: python
+
+   from determined import pytorch
+   def main():
+       with det.pytorch.init() as train_context:
+           trial = MyTrial(train_context)
+           trainer = det.pytorch.Trainer(trial, train_context)
+   +       trainer.fit(
+   +           checkpoint_period=pytorch.Batch(100),
+   +           validation_period=pytorch.Batch(100),
+   +           checkpoint_policy="all"
+   +       )
+
+   if __name__ == "__main__":
+       # Configure logging
+       logging.basicConfig(level=logging.INFO, format=det.LOG_FORMAT)
+       main()
+
+Run Your Training Script Locally
+================================
+
+Run training scripts locally without submitting to a cluster or defining an experiment configuration
+file. Be sure to specify ``max_length`` in the ``.fit()`` call, which is used in local training mode
+to determined the maximum number of steps to train for.
+
+.. code:: python
+
+   from determined import pytorch
+
+
+   def main():
+       with det.pytorch.init() as train_context:
+           trial = MyTrial(train_context)
+           trainer = det.pytorch.Trainer(trial, train_context)
+           trainer.fit(
+               +max_length=pytorch.Epoch(1),
+               checkpoint_period=pytorch.Batch(100),
+               validation_period=pytorch.Batch(100),
+               checkpoint_policy="all",
+           )
+
+
+   if __name__ == "__main__":
+       # Configure logging
+       logging.basicConfig(level=logging.INFO, format=det.LOG_FORMAT)
+       main()
+
+You can run this Python script directly (``python3 train.py``), or in a Jupyter notebook. This code
+will train for one epoch, and checkpoint and validate every 100 batches.
+
+Local Distributed Training
+==========================
+
+Local training can utilize multiple GPUs on a single node with a few modifications to the above
+code.
+
+.. note::
+
+   Both Horovod and PyTorch Distributed backends are supported.
```

Comment:
I don't think this is interesting enough for a `.. note` admonition.  I think if we overuse those visual cues, we reduce their effectiveness.